pages/404.tsx

```diff
@@ -0,0 +1,16 @@
+// 404.js
+
+import Link from "next/link";
+
+export default function FourOhFour() {
+  return (
+    <>
+      <h1>404 - Page Not Found</h1>
+      We&apos;re very sorry - we could not find the page you were looking for.
+      Please navigate to the Teleport <Link href="/docs/">
+        Documentation
+      </Link>{" "}
+      or the <a href="/">Home Page</a> to find what you&apos;re looking for.
```

Comment:
I know this is techincally correct, but this sentence reads weird to me. "We couldn't find what you're looking for so go here to find what you're looking for" . Seems odd. NOT a blocker, but maybe we can revisit the language? idk

Reply:
I'd just changed the file's extension in this PR without changing the content, so I'd propose we do it in a separate PR.